pkg/asset/operators/utils.go

```diff
@@ -0,0 +1,59 @@
+package operators
+
+import (
+	"github.com/ghodss/yaml"
+	"github.com/openshift/installer/pkg/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type configurationObject struct {
+	metav1.TypeMeta
+
+	Metadata metadata    `json:"metadata,omitempty"`
+	Data     genericData `json:"data,omitempty"`
+}
+
+type metadata struct {
+	Name      string `json:"name,omitempty"`
+	Namespace string `json:"namespace,omitempty"`
+}
+
+func configMap(namespace, name string, data genericData) (string, error) {
+	configurationObject := configurationObject{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "ConfigMap",
+		},
+		Metadata: metadata{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Data: data,
+	}
+
+	str, err := marshalYAML(configurationObject)
+	if err != nil {
+		return "", err
+	}
+	return str, nil
+}
+
+func marshalYAML(obj interface{}) (string, error) {
```

Comment:
this seems to be a function that us already present in some common package?

Reply:
Ack.
Maybe all of this file's contents can be moved out. This is the common utils package being born.